code/frameworks/nextjs/package.json

```diff
@@ -0,0 +1,129 @@
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./preset": {
+      "require": "./dist/preset.js",
+      "import": "./dist/preset.mjs",
+      "types": "./dist/preset.d.ts"
+    },
+    "./preview.js": {
+      "require": "./dist/preview.js",
+      "import": "./dist/preview.mjs",
+      "types": "./dist/preview.d.ts"
+    },
+    "./next-image-loader-stub.js": {
+      "require": "./dist/next-image-loader-stub.js",
+      "import": "./dist/next-image-loader-stub.mjs",
+      "types": "./dist/next-image-loader-stub.d.ts"
+    },
+    "./resolved-router-context.js": {
+      "require": "./dist/resolved-router-context.js",
+      "import": "./dist/resolved-router-context.mjs",
+      "types": "./dist/resolved-router-context.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "types/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "check": "tsc --noEmit",
+    "prepare": "../../../scripts/prepare/bundle.ts"
+  },
+  "dependencies": {
+    "@storybook/addons": "7.0.0-alpha.35",
+    "@storybook/builder-webpack5": "7.0.0-alpha.35",
+    "@storybook/core-common": "7.0.0-alpha.35",
+    "@storybook/node-logger": "7.0.0-alpha.35",
+    "@storybook/preset-react-webpack": "7.0.0-alpha.35",
+    "@storybook/react": "7.0.0-alpha.35",
+    "@types/node": "^14.14.20 || ^16.0.0",
+    "fs-extra": "^9.0.1",
+    "image-size": "^1.0.0",
+    "loader-utils": "^3.2.0",
+    "pnp-webpack-plugin": "^1.7.0",
+    "postcss-loader": "^6.2.1",
+    "resolve-url-loader": "^5.0.0",
+    "sass-loader": "^12.4.0",
+    "semver": "^7.3.5",
+    "tsconfig-paths": "^4.0.0",
+    "tsconfig-paths-webpack-plugin": "^3.5.2"
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "7.0.0-alpha.35",
+    "@types/loader-utils": "^2.0.3",
+    "next": "^12.2.4",
+    "nextv10": "npm:next@10.0.0",
+    "nextv11": "npm:next@11.0.0",
+    "nextv9": "npm:next@9.0.0",
+    "typescript": "~4.6.3",
+    "webpack": "^5.65.0"
+  },
+  "peerDependencies": {
+    "@babel/core": "^7.11.5",
+    "@storybook/addon-actions": "7.0.0-alpha.35",
+    "next": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0",
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "webpack": "^5.0.0"
+  },
+  "peerDependenciesMeta": {
+    "@babel/core": {
+      "optional": true
+    },
+    "@storybook/addon-actions": {
+      "optional": true
+    },
+    "typescript": {
+      "optional": true
+    }
+  },
+  "engines": {
+    "node": ">=10.13.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bundler": {
+    "entries": [
+      "./src/index.ts",
+      "./src/preset.ts",
+      "./src/preview.tsx",
+      "./src/next-image-loader-stub.ts",
+      "./src/resolved-router-context.ts"
+    ],
+    "platform": "node"
+  }
+}
```

Comment:
Other frameworks have a `gitHead` property. Is that auto-generated by some release process?